MekHQ/src/mekhq/campaign/CampaignOptions.java

```diff
@@ -2826,8 +2774,8 @@ public static CampaignOptions generateCampaignOptionsFromXml(Node wn) {
                 retVal.useQualityMaintenance = Boolean.parseBoolean(wn2.getTextContent());
             } else if (wn2.getNodeName().equalsIgnoreCase("reverseQualityNames")) {
                 retVal.reverseQualityNames = Boolean.parseBoolean(wn2.getTextContent());
-            } else if (wn2.getNodeName().equalsIgnoreCase("useUnofficalMaintenance")) {
```

Comment:
Not sure what happened here, I'll double check.

Reply:
So this line is added in line 2734 and 2740. It's just earlier because refactoring compressed this method. I have double checked in the file and the code to write/read this variable is there.

Reply:
I'm referring to the old `useUnofficalMaintenance`, you'll need to support reading that *and* the new correctly spelled `useUnofficialMaintenance`.

Reply:
Oh, I totally missed that. Do we want to have to deal with that? I could just revert the rename.

Reply:
As long as an old campaign gets carried forward fine, I'm ok with either adding the old name too or reverting the node name changes.

Reply:
I reverted the change on the XML, but addressed the typo on code (methods, variables).